Added the Weights and Condition scores schemas

AndyDe-Aldent opened the pull request.

AndyDe-Aldent committed:
Merge pull request #2 from integritysystemscompany/master

merging current state of main repo to fork

Commit:
added the Weights and condition scores schema

anitanagaraj commented on JSON_Schemas/Weights_and_Condition_Scores/iscFatScore.json:
```diff
@@ -0,0 +1,46 @@
+                            "fatDepth": {
+                                "type": "number",
+                                "format": "double",
+                                "description": "Usually measure in Millimeters"
```
Description - usually "measured'' in Millimeters

cookeac replied:
I thought we had agreed that Fat Depth was not part of fat score and was in fact a separate measure and not a score at all.

AndyDe-Aldent replied:
Removed the fat depths from iscFatScore

anitanagaraj commented on JSON_Schemas/Weights_and_Condition_Scores/iscFrameScore.json:
```diff
@@ -0,0 +1,29 @@
+                },
+                "score":{
+                    "type": "number",
+                    "description": "Value of the frame score"
```
Add to description - Frame score is a  convenient way to discuss and evaluate height is in units of 'frame score' based on the height over the hips at a given age

AndyDe-Aldent replied:
Added the description

anitanagaraj commented on JSON_Schemas/Weights_and_Condition_Scores/iscMuscleScore.json:
```diff
@@ -0,0 +1,30 @@
+                "score":{
+                    "type": "string",
+                    "enum": ["A", "B", "C", "D", "E"],
+                    "description": "Value of the muscle score"
```
Add to description - Muscle score measures the degree of muscularity of an animal and describes the shape of cattle independent of the influence of fatness.

AndyDe-Aldent replied:
Added to the description

anitanagaraj commented on JSON_Schemas/Weights_and_Condition_Scores/iscConditionScore.json:
```diff
@@ -0,0 +1,29 @@
+                },
+                "score":{
+                    "type": "number",
+                    "description": "Value of the condition score"
```
Add to description - Body condition scoring is a visual assessment of the amount of fat and muscle covering the bones of a cow, regardless of body size

AndyDe-Aldent replied:
Added description.

cookeac commented on JSON_Schemas/Weights_and_Condition_Scores/iscScoreEventCore.json:
```diff
@@ -0,0 +1,39 @@
+                    "type": "object",
+                    "$ref": "iscMuscleScore",
+                    "description": "Event object for capturing Muscle Score details"
+                }
```
I thought the score core event was to provide for assessor, and there would be individual score events for each of the types, rather than building them into iscScoreEventCore??

AndyDe-Aldent replied:
Change is done in the current version of the schema

cookeac commented on JSON_Schemas/Weights_and_Condition_Scores/iscWeightEventResource.json:
```diff
@@ -0,0 +1,53 @@
+                    "type": "string",
+                    "enum": ["KGM", "GRM", "LBR", "TNE", "MC", "MGM"],
+                    "default": "KGM",
+                    "description": "Units specified in UN/CEFACT 3-letter form. Default if not specified is KGM. Mass units for weight from UN/CEFACT trade facilitation recommendation 20.\r\n Kilogram, Gram, Pound, Metric Tonne, Microgram, Miligram."
```
Any reason why we are redefining these here rather than using the uncefactMassUnitsType as per the ICAR definition?

AndyDe-Aldent committed:
Update iscScoreEventCore.json

removed the extra fields from the iscScoreEventCore

AndyDe-Aldent committed:
Update iscWeightEventResource.json

fixed issues in the iscWeightEventResource.Json

AndyDe-Aldent committed:
Create iscDeviceReferentType.json

AndyDe-Aldent committed:
Rename iscDeviceReferentType.json to iscDeviceReferenceType.json

AndyDe-Aldent committed:
Merge pull request #3 from integritysystemscompany/master

merging files back from main repository

Commit:
Added the JSON files for the Health Treatment schema

AndyDe-Aldent committed:
Merge pull request #4 from integritysystemscompany/master

merge changes to spreadsheet to fork

AndyDe-Aldent committed:
Update iscDiagnosisType.json

Changed from using a venom code over to an iscDiagnosisIdentifierType for the diagnosis code

AndyDe-Aldent committed:
Create iscDiagnosisIdentifierType.json

AndyDe-Aldent committed:
Update iscFatScore.json

removed fat depths

AndyDe-Aldent committed:
Update iscConditionScore.json

AndyDe-Aldent committed:
Update iscMuscleScore.json

AndyDe-Aldent committed:
Update iscFrameScore.json
